airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/AbstractSshMssqlSourceAcceptanceTest.java

```diff
@@ -49,7 +51,7 @@ protected JsonNode getConfig() {
 
   @Override
   protected void setupEnvironment(final TestDestinationEnv environment) throws Exception {
-    testdb = MsSQLTestDatabase.in("mcr.microsoft.com/mssql/server:2017-latest", "withNetwork");
+    testdb = MsSQLTestDatabase.in(BaseImage.MSSQL_2017_LATEST, ImageModifier.NETWORK);
```

Comment:
These MS SQL Server changes seem to have made their way into this PR by mistake.